### ISSUE TITLE
[Issue] Npm scripts failing to run correctly

### ISSUE THREAD

**Issue by Cellv2** [cmderdev/cmder#1596]
### Version

I have replicated this issue on Cmder versions 1.3.1 and 1.3.4.

### Issue:
When running cmder, certain npm scripts (or combination of scripts) do not seem to run correctly. In my particular case, I am running both an Express server and json-server, both of which require ports to be opened. I am using npm-run-all to run these in parallel, but the json-server port does not get opened.

I tried reversing the ports (so Express uses 3001 and json-server uses 3000), but the issue remained. I have also tried running a new console as an admin, but the issue is still there.

I believe this is an issue with cmder rather than my project because:
- I'm able to replicate this problem on multiple machines, running different versions of Node/npm, as well as multiple separate projects (though the issue still appears to be down to trying to open two ports at once).
- I do not have the issue if I use Windows CMD.

### Example:
A repo that I can replicate this on can be found [here](https://github.com/Cellv2/js-dev-env). I'm using old dependency versions here, but it didn't seem to make any difference when I tested upgrading them.

In order to replicate the issue, download the project, npm install and then npm start. This should start both Express and json-server. In cmder, however, port 3001 does not appear to open, stopping json-server from serving the JSON file under src/api/db.json .

If you run _npm run start-mockapi_, then json-server does start, but it does not appear to start when combined with something else that opens a port (at least that's what I take from it). 

### Workaround:
Other than using Windows CMD, I've found two workarounds, neither of which I can make sense of 😟 

The first is to simply add _start-mockapi_ twice into the start script under package.json ("start": "run-p security-check open:src lint:watch test:watch start-mockapi **start-mockapi**"). Once updated and saved, json-server appears to run without issue.

The second is to remove _start-mockapi_ from the start script ("start": "run-p security-check open:src lint:watch test:watch"), then in the console run _npx run-p start-mockapi start_ - running both start-mockapi and start in parallel - which appears to make json-server run without issue as well.

If you need any further information at all, I'm happy to provide any relevant information. Hopefully it's just me being silly 😃

**Comment by Cellv2** [comment]
I suspect this is an issue with ConEmu. Closing this issue as I've raised a separate issue on the ConEmu page (issue 1370).
